src/components/plots/xyPlot.ts

```diff
@@ -27,8 +27,8 @@ module Plottable {
       }
       this.classed("xy-plot", true);
 
-      this._xScale = xScale;
-      this._yScale = yScale;
+      this._propertyBindings.set(XYPlot._X_KEY, { accessor: null, scale: xScale });
+      this._propertyBindings.set(XYPlot._Y_KEY, { accessor: null, scale: yScale});
```

Comment:
Cause the domainers are set for them immediately afterward.

I think this would be a lot more clear if scales were not set here and have the constructor be light